runtime/src/group.rs

```diff
@@ -0,0 +1,392 @@
+//! # Group Module
+//!
+//! The Group module is used to manage a set of identities. A group of identities can be a
+//! collection of KYC providers, council members for governance and so on. This is an instantiable
+//! module.
+//!
+//! ## Overview
+//! Allows control of membership of a set of `IdentityId`s, useful for managing membership of a
+//! collective.
+//!
+//! - Add a new identity
+//! - Remove identity from the group
+//! - Swam members
+//! - Reset group members
+//!
+//! ### Dispatchable Functions
+//!
+//! - `add_member` - Adds a new identity to the group.
+//! - `remove_member` - Remove identity from the group if it exists.
+//! - `swap_member` - Replace one identity with the other.
+//! - `reset_members` - Re-initialize group members.
+//!
```

Comment:
Since this is the modified version of the membership module, we should include the relevant license and context at the top.

Reply:
@adamdossa I'll let you make a call on whether or not we need license for this module. This module is definitely inspired by substrate's `membership` module and the dispatchables take the similar function signatures. However the context and intent is different and it isn't just "extending" `membership` module by adding functions to it like we did with `staking`.

Reply:
I will check w/ Jesse what the appropriate approach is.